system-tests/005/__snapshots__/005.test.js.snap

```diff
@@ -11765,169 +11765,169 @@ Array [
         "text": "Expected line length to be no more than 20 characters (max-line-length)",
       },
       Object {
-        "column": 5,
-        "line": 146,
+        "column": 25,
+        "line": 145,
```

Comment:
Something seems wrong with this system test. Am I right that these line numbers do not correspond with lines in `005/stylesheet.css` that make sense? Let's sort this out before accepting changes to it.